api4/team.go

```diff
@@ -58,6 +60,8 @@ func (api *API) InitTeam() {
 	api.BaseRoutes.Team.Handle("/invite/email", api.ApiSessionRequired(inviteUsersToTeam)).Methods("POST")
 	api.BaseRoutes.Teams.Handle("/invites/email", api.ApiSessionRequired(invalidateAllEmailInvites)).Methods("DELETE")
 	api.BaseRoutes.Teams.Handle("/invite/{invite_id:[A-Za-z0-9]+}", api.ApiHandler(getInviteInfo)).Methods("GET")
+
+	api.BaseRoutes.Teams.Handle("/{team_id:[A-Za-z0-9]+}/if_groups_then_users_removed", api.ApiSessionRequired(ifGroupsThenUsersRemoved)).Methods("GET")
```

Comment:
the url is a bit confusing to me, I don't think is a clear name. maybe `users_removed_on_changed_groups` `users_removed_if_groups_change`. I think is more clear an explicit.

Reply:
When you think about it doesn't matter if any hypothetical case is going to happens; that has no bearing on these results now. What this really is is the difference between the **set of team members** and the **set of given group members**, expressed as `Users` records. I'm going to renaming everything with that concept in mind. Perhaps:

`/teams/:team_id/members_minus_group_members`
and
`TeamMembersMinusGroupMembers`

Reply:
Yes, that is very explicit about what it does. I like it.